### PR TITLE
Refactor add employee

### PR DESCRIPTION
**Pull request by andreeaadm1408**
-add role
-insert encrypted password in the database

### CODE REVIEW

File: src/main/java/ro/db/vendor/service/EmployeesServiceImpl.java [hunk 1]
```diff
@@ -86,13 +98,20 @@ public Employees save(EmployeeModelForAdd employeeModel){
       Departments department = new Departments();
       department.setIdDepartament(employeeModel.getId_departament());
 
+      Roles role;
+      if(employeeModel.isIs_mng()) {
+        role = roleService.findRoleById(2);
```
Comment (alexandrubirlan) [inline src/main/java/ro/db/vendor/service/EmployeesServiceImpl.java]: use the roles enum

File: src/main/java/ro/db/vendor/service/EmployeesServiceImpl.java [hunk 2]
```diff
@@ -86,13 +98,20 @@ public Employees save(EmployeeModelForAdd employeeModel){
...
+      }else {
+        role = roleService.findRoleById(3);
```
Comment (alexandrubirlan) [inline src/main/java/ro/db/vendor/service/EmployeesServiceImpl.java]: use the roles enum

File: pom.xml [hunk 3]
```diff
@@ -135,5 +135,15 @@
             <version>${spring.version}</version>
             <scope>compile</scope>
         </dependency>
+      <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-core</artifactId>
+        <version>5.0.4.RELEASE</version>
```
Comment (alexandrubirlan) [inline pom.xml]: use right version here

**Comment by andreeaadm1408** [comment]
I created another branch ( https://github.com/iancdiana21/dbVendor/pull/44)
I tested the save method with mocks.
The Roles enum it's already implemented and committed on master.

**Comment by AdyT14** [comment]
This PR was completed during another one and i will close it.

**Pull request closed without merging**